canary/consumer-java/src/main/java/com/amazon/kinesis/video/canary/consumer/CanaryFragmentList.java

```diff
@@ -0,0 +1,27 @@
+package com.amazon.kinesis.video.canary.consumer;
+
+import java.util.List;
+import java.util.ArrayList;
+
+
+/*
+    CanaryFragmentList allows for intervalMetricsTask calls to access the same data by passing a CanaryFragmentList object.
+ */
+
+public class CanaryFragmentList {
+
+    private List<CanaryFragment> fragmentList = new ArrayList<>();
```

Comment:
As a convention, it is preferable to prefix the class variable names with m. Can we have this change throughout wherever relevant?
`fragmentList` -> `mFragmentList`

Reply:
Updated. I didn't do this for CanaryListFragmentWorker as I'm trying to keep the code as similar to the example it's modeled after as possible.